benches/match.rs

```diff
@@ -273,7 +273,7 @@ fn criterion_word100(c: &mut Criterion) {
         b.iter(|| {
             let mut x = 0;
             for s in &word_100 {
-                trie_match!(match s {
+                trie_match!(match s.as_str() {
```

Comment:
It looks this breaks some backward compatibility but the description of the PR doesn't contain any explanation.

Reply:
I updated the description and increments the version number to 0.2.0